### DISCUSSION TITLE
Git Repo Name

### DISCUSSION THREAD

**Discussion by Jasonlovedataanalyst** [CarlosPietrasanta/Tafe230130#3] [Q&A]
@ozdotcom @CarlosPietrasanta 

Hello Morning 

I just came across this part in the instruction of assignment 2, should we consult with the teacher for the repo's name 

**** quote from the assignment instruction documents 

_> Setting up the local repository:
> 
> Ø Creates a local working directory and create a local Git repository called Tafexxxxyy. (The name will be allocated by the lecturer)._

**Comment by CarlosPietrasanta** [comment]
Hi Jason, 

Sorry we didn't tell you about that before, but the name for the repo was already assigned by the lecturer when he created the groups originally, in an email he sent for external students. Don't have to worry about the name, I created the repo using the provided instructions, but thanks for keeping an eye on it.

<img width="1173" alt="Untitled" src="https://user-images.githubusercontent.com/127190272/226503584-cd4baff8-b651-4328-80b6-9810fbeb711c.png">

**Reply by ozdotcom** [reply]
Thanks for your quick response to Jason, Carlos.
